lib/we/call/middleware.rb

```diff
@@ -1,7 +1,6 @@
 module We
   module Call
     module Middleware
-      autoload :Client, 'we/call/middleware/client'
```

Comment:
We don't have any client middlewares right now. If we make some, this'll come back.